cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -2031,6 +2161,52 @@ describe('CourseOutlinePage', function() {
             );
         });
 
+        it('due num weeks (due_in) can be cleared.', function() {
+            createCourseOutlinePage(this, mockCourseJSON, false);
+            setSelfPacedCustom();
+            outlinePage.$('.outline-item .outline-subsection .configure-button').click();
+            setEditModalValuesForCustomPacing('3', 'Lab');
+            setContentVisibility('staff_only');
+            $('.wrapper-modal-window .action-save').click();
+
+            // This is the response for the change operation.
+            AjaxHelpers.respondWithJson(requests, {});
+            // This is the response for the subsequent fetch operation.
+            AjaxHelpers.respondWithJson(requests, mockCustomPacingServerValuesJson);
+
+            expect($('.outline-subsection .status-grading-value')).toContainText(
+                'Lab'
+            );
+            expect($('.outline-subsection .status-message-copy')).toContainText(
+                'Contains staff only content'
+            );
+
+            outlinePage.$('.outline-subsection .configure-button').click();
+            expect($('#due_in').val()).toBe('3');
+            expect($('#grading_type').val()).toBe('Lab');
+            expect($('input[name=content-visibility][value=staff_only]').is(':checked')).toBe(true);
+
+            $('.wrapper-modal-window .due-date-input .action-clear').click();
+            expect($('#due_in').val()).toBe('');
+
+            $('#grading_type').val('notgraded');
+            setContentVisibility('visible');
+
+            $('.wrapper-modal-window .action-save').click();
+
+            // This is the response for the change operation.
+            AjaxHelpers.respondWithJson(requests, {});
+            // This is the response for the subsequent fetch operation.
+            AjaxHelpers.respondWithJson(requests,
+                createMockSectionJSON({}, [createMockSubsectionJSON()])
+            );
+            expect($('.outline-subsection .status-grading-date')).not.toExist();
+            expect($('.outline-subsection .status-grading-value')).not.toExist();
```

Comment:
Are these copy pasted over? It looks like you're wanting to test here that the value is till empty after saving, but these are from the instructor paced tests.

Reply:
Yup this is the same piece of code from the test case right above -- I should have been more specific with my test case description, but I didn't think it was a bad idea to also make sure the other information in the outline is also cleared out along with the input for `due_in` in the modal.